### DISCUSSION TITLE
помогите с хостингом

### DISCUSSION THREAD

**Discussion by pikni4okkk** [Rapptz/discord.py#9635] [Q&A]
у меня есть хостинг dscrd и на нем стоит мой бот для дискорда, я написал код, но для него нужна библиотека discord.py, как мне ее добавить/загрузить на этот хостинг?

**Comment by No767** [comment]
For hosting, you want to use an VPS. For more details on this, it would be advisable to join the [discord server](https://discord.gg/r3sSKJJ) instead
